po/de.po

```diff
@@ -1542,135 +1626,140 @@ msgid ""
 msgid "GTG was made by many contributors around the world."
-msgstr ""
+msgstr "GTG wurde von vielen Beitragende weltweit mitgewirkt."
 
-#: GTG/gtk/browser/main_window.py:309
+#: GTG/gtk/browser/main_window.py:310
 msgid "The GTG project is maintained/administered by:"
-msgstr ""
+msgstr "Das GTG Projekt wird verwaltet und betreut von:"
 
-#: GTG/gtk/browser/main_window.py:311
+#: GTG/gtk/browser/main_window.py:312
 msgid "This release was brought to you by the efforts of these people:"
 msgstr ""
+"Diese Version wurden durch die Anstrengungen folgende Beitragenden möglich:"
 
-#: GTG/gtk/browser/main_window.py:313
+#: GTG/gtk/browser/main_window.py:314
 #, python-brace-format
 msgid ""
 "Many others contributed to GTG over the years.\n"
 "You can see them on {OH_stats} and {GH_stats}."
 msgstr ""
+"Über die Jahre haben viele GTG etwas beigetragen.\n"
+"Unter {OH_stats} und {GH_stats} sind sie zu finden."
 
-#: GTG/gtk/browser/main_window.py:321
+#: GTG/gtk/browser/main_window.py:322
 msgid "GTG website"
-msgstr "Getting-Things-Gnome!-Webseite"
+msgstr "Webseite von GTG"
 
+# Borim translated before but has not contact info nor is being credited, so adding just in case
 #. Translators for a particular language should put their names here.
 #. Please keep the width at 80 chars max, as GTK3's About dialog won't wrap text.
 #. GtkAboutDialog will detect if “translator-credits” is untranslated and auto-hide the tab.
-#: GTG/gtk/browser/main_window.py:334
+#: GTG/gtk/browser/main_window.py:335
 msgid "translator-credits"
 msgstr ""
+"Borim\n"
+"Neui"
 
-#: GTG/gtk/browser/main_window.py:636
-#, fuzzy
+#: GTG/gtk/browser/main_window.py:637
 msgid "Hide Sidebar"
-msgstr "_Schlagwörter-Seitenleiste"
+msgstr "Seitenleiste verstecken"
 
-#: GTG/gtk/browser/main_window.py:638
-#, fuzzy
+#: GTG/gtk/browser/main_window.py:639
 msgid "Show Sidebar"
-msgstr "_Schlagwörter-Seitenleiste"
+msgstr "Seitenleiste anzeigen"
 
 #. Rember values from last time
 #: GTG/gtk/browser/modify_tags.py:40
 msgid "NewTag"
 msgstr "Neues Schlagwort"
 
 #: GTG/gtk/browser/simple_color_selector.py:237
-#, fuzzy
 msgid "Add Custom Color..."
-msgstr "Benutzerdefinierte Farbe hinzufügen"
+msgstr "Benutzerdefinierte Farbe hinzufügen..."
 
 #: GTG/gtk/browser/simple_color_selector.py:242
-#, fuzzy
 msgid "Clear Selected Color"
-msgstr "Entferne ausgewähltes Symbol."
+msgstr "Entferne ausgewählte Farbe"
 
 #: GTG/gtk/browser/simple_color_selector.py:283
 msgid "Choose a color"
 msgstr "Farbe auswählen"
 
 #: GTG/gtk/browser/tag_context_menu.py:60
-#, fuzzy
 msgid "Generate Color"
-msgstr "Farben zurücksetzen"
+msgstr "Farben generieren"
 
 #: GTG/gtk/browser/tag_editor.py:80
 msgid "Remove selected icon"
-msgstr "Entferne ausgewähltes Symbol."
+msgstr "Entferne ausgewähltes Symbol"
 
 #: GTG/gtk/browser/tag_editor.py:199
-#, fuzzy, python-format
+#, python-format
 msgid "Editing Tag \"%s\""
-msgstr "Aufgabe hinzufügen"
+msgstr "Editiere Schlagwort \"%s\""
```

Comment:
"Bearbeite" instead of "Editiere".
See https://wiki.gnome.org/de/UebersetzungsRichtlinien